try
  clear a
  a;
catch
end_try_catch
1
